hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestHoodieReadClient.java

```diff
@@ -50,16 +48,7 @@
 @SuppressWarnings("unchecked")
 public class TestHoodieReadClient extends HoodieClientTestBase {
 
-  @Override
-  protected void initPath() {
-    try {
-      java.nio.file.Path basePath = tempDir.resolve("dataset");
-      java.nio.file.Files.createDirectories(basePath);
-      this.basePath = basePath.toUri().toString();
-    } catch (IOException ioe) {
-      throw new HoodieIOException(ioe.getMessage(), ioe);
-    }
-  }
+  private static final int PARALLELISM = 2;
```

Comment:
Was wondering if this can be paramterized as a maven option? Would be helpful when we run tests on bigger cluster.

Reply:
@codope i've filed this https://issues.apache.org/jira/browse/HUDI-5287

in principle we should make UT/FT only run against small number of records and runnable even on cheap machines. So we should be able to keep parallelism low and fixed. We only make num records and parallelism configurable when running ITs with varying number of records.